database/migrations/2021_07_22_080405_create_dictionary_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateDictionaryTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('dictionaries', function (Blueprint $table) {
+            $table->id();
+            $table->string("key")->unique();
```

Comment:
imo w takich miejscach fajnie mieć nawyk dawania nowej linii czyli:
$table->string("key")
    ->unique();
Bo jak dodaje się więcej rzeczy to taga giga długa linijka powstaje i średnio się to czyta

Reply:
No, nie formatuje mi formatter tak :( Muszę ustawić go, żeby to robił, jak się da i w sumie fajnie by było dodać to do lintera, żeby tez takie coś sprawdzał, jak doda ktoś lintera do actions.